### PR TITLE
mckie-feature-5

### PR DESCRIPTION
**Pull request by mckiekun**

### CODE REVIEW

File: application/controllers/TodoController.php [hunk 1]
```diff
@@ -50,11 +50,26 @@ public function data_Table(){
...
+				 'age'=>$this->input->post('age', TRUE),
+				 'id'=>$this->input->post('id', TRUE),
+				 'row_status'=> 0
+					];
+	
+		
+		//$this->load->model('PlayerModel');
+		$deletedata = $this->PlayerModel->saverecords($data);
+		echo $deletedata;
+	}
```
Comment (amobyJD) [inline application/controllers/TodoController.php:67]: EXCELLENT

**Comment by amobyJD** [comment]
dami mong issue mck

File: application/controllers/TodoController.php [hunk 2]
```diff
@@ -51,18 +51,24 @@ public function data_Table(){
...
-				 'age'=>$this->input->post('age', TRUE),
-				 'id'=>$this->input->post('id', TRUE),
-				 'row_status'=> 0
-					];
+		// $data = ['first_name'=>$this->input->post('first_name', TRUE),
+		// 		 'last_name'=>$this->input->post('last_name', TRUE),
+		// 		 'age'=>$this->input->post('age', TRUE),
+		// 		 'id'=>$this->input->post('id', TRUE),
+		// 		 'row_status'=> 0
+		// 			];
```
Comment (amobyJD) [inline application/controllers/TodoController.php]: please remove this.. useless codes

File: application/controllers/TodoController.php [hunk 3]
```diff
@@ -51,18 +51,24 @@ public function data_Table(){
...
 	
 		
-		//$this->load->model('PlayerModel');
-		$deletedata = $this->PlayerModel->saverecords($data);
-		echo $deletedata;
+		// //$this->load->model('PlayerModel');
+		// $deletedata = $this->PlayerModel->saverecords($data);
+		// echo $deletedata;
```
Comment (amobyJD) [inline application/controllers/TodoController.php]: i think this also useless

File: assets/js/modules/todo-module.js [hunk 4]
```diff
@@ -96,6 +96,36 @@ Todo = function () {
...
+		// 					}
+		// 				}
+		// 			});
+		// 		}
+		// 	});
+		// 	//return false;
+		// }
+		
+	
+
```
Comment (amobyJD) [inline assets/js/modules/todo-module.js:102]: what is this?

**Review by amobyJD — CHANGES_REQUESTED** [review]

File: application/controllers/TodoController.php (hunk 2 above)
Comment (mckiekun) [reply application/controllers/TodoController.php]: k

**Review by amobyJD — APPROVED** [review]
Well done @mckiekun you may now study the iPRO

**Pull request merged**